app/src/main/java/com/github/se/travelpouch/model/travels/TravelRepositoryMock.kt

```diff
@@ -0,0 +1,68 @@
+package com.github.se.travelpouch.model.travels
+
+import android.util.Log
+import com.github.se.travelpouch.model.profile.Profile
+import com.github.se.travelpouch.travelCollection
+
+class TravelRepositoryMock : TravelRepository {
+
+  private var currentUserUid = ""
+
+  override fun getNewUid(): String {
+    return TravelContainerMock.generateAutoObjectId()
+  }
+
+  override fun getParticipantFromfsUid(
+      fsUid: fsUid,
+      onSuccess: (Profile?) -> Unit,
+      onFailure: (Exception) -> Unit
+  ) {
+    TODO("Not yet implemented")
```

Comment:
Technically these functions are not used so no need to implement them. Later, we will have to implenment them. But for new they are not usefull (and I cannot remove  them since they come from an interface the class implements)